x-pack/test/functional_with_es_ssl/apps/cases/group2/attachment_framework.ts

```diff
@@ -182,14 +183,14 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
       });
 
       it('renders multiple attachment types correctly', async () => {
-        const theCase = await getCase({
+        const response = await getAllUserActions({
```

Comment:
getAllComments didn't work anymore, couldn't come up with a simpler way of retrieving the comments here. The fake route we added for other integration test seems to not run here

Reply:
What about using the `findAttachments` or `getAllComments` helper function from `x-pack/test/cases_api_integration/common/lib/api/attachments.ts`?

Reply:
`findAttachments` doesn't return any comment and `getAllComments` is 404 because our fake route is not loaded in this test type